src/catkin_pkg/changelog_generator_vcs.py

```diff
@@ -146,11 +146,11 @@ def _get_author(self, hash_):
         return result['output']
 
     def get_tags(self):
-        cmd_tag = [self._executable, 'tag']
+        cmd_tag = [self._executable, 'log', '--simplify-by-decoration', '--pretty="%d"']
```

Comment:
Since the output contains more than tag names I am worried about parts of it being locale specific. Can you try it with a different locale and check if `tag` is still called `tag`? If not, the command needs to be run with a fixed locale like `LANG=en_US.UTF-8`.

The double quotes around `%d` are not necessary.